### ISSUE TITLE
Overflow handling/free list for entity Ids

### ISSUE THREAD

**Issue by ashoulson** [ashoulson/RailgunNet#10]

**Comment by ashoulson** [comment]
Probably need to do this by keeping a free list. When an entity removal event goes out and all clients have acked it, it's safe to retire that Id.

What about clients that leave before the remove event is acked?

**Comment by ashoulson** [comment]
Depends on #8

**Comment by ashoulson** [comment]
No longer relevant now that we can elastic-encode uints. Maximum tick and Entity ID will be 4,294,967,295, at the cost of 5 bytes. Not expecting to ever hit that. That's 2.5 years at 50Hz.
